__tests__/app.test.js

```diff
@@ -183,3 +183,46 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("200: PATCH /api/articles/:article_id should respond with an updated article with votes increased/decreased by provided value", () => {
+    const votes = { inc_votes: 5 };
+    return request(app)
+      .patch("/api/articles/4")
+      .send(votes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article[0].votes).toBe(5);
+        expect(body.article[0]).toMatchObject({
```

Comment:
This article should not be an array but an object, since we are only expecting one article back it should be the object itself we receive.